services/azure-storage/src/provide_credential/azure_cli.rs

```diff
@@ -96,12 +96,12 @@ impl ProvideCredential for AzureCliCredentialProvider {
 
         // Calculate expiration time
         let expires_on = if let Some(timestamp) = token.expires_on_timestamp {
-            Some(chrono::DateTime::from_timestamp(timestamp, 0).unwrap())
+            Some(jiff::Timestamp::new(timestamp, 0).unwrap())
         } else if let Some(expires_str) = token.expires_on {
             // Parse the string format "2023-10-31 21:59:10.000000"
-            chrono::NaiveDateTime::parse_from_str(&expires_str, "%Y-%m-%d %H:%M:%S%.f")
+            jiff::civil::DateTime::strptime("%Y-%m-%d %H:%M:%S%.f", &expires_str)
+                .and_then(|dt| jiff::tz::TimeZone::UTC.to_timestamp(dt))
```

Comment:
This makes me somewhat sad. I realize this is what you were doing before, but assuming UTC for datetime string without any time zone or offset is kind of a bummer. It can be rather ambiguous. It's too easy for users to assume the wrong thing (e.g., a local time).

Reply:
cc @Xuanwo is this an issue from Azure Storage that they explicitly declare the expires_on string is of UTC but returned without a timezone?

Reply:
https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest#refresh-tokens